eos-phone-home

```diff
@@ -321,42 +321,36 @@ class PhoneHome(object):
         for name in variables:
             request[name] = self._lookup_or_get_variable(name)
 
-        request_data = json.dumps(request)
-        print("Request data:", request_data)
+        log.info("Request data: %s", request)
 
-        print("Sending to server (%s)..." % endpoint)
+        log.info("Sending to server (%s)...", endpoint)
 
         if self._debug:
-            print("Debugging turned on so nothing sent to server!")
+            log.info("Debugging turned on so nothing sent to server!")
             return True
 
-        success = False
-
         try:
-            data = request_data.encode('utf-8')
-
-            req = urllib.request.Request(endpoint, data=data, method='PUT')
-            req.add_header('Content-Type', 'application/json')
-
-            response_data = None
-            with urllib.request.urlopen(req) as conn:
-                response_data = conn.read().decode('utf-8')
-
-            print('Response:', response_data)
-
-            response = json.loads(response_data)
+            response = requests.put(endpoint, json=request)
+            response.raise_for_status()
+        except requests.exceptions.RequestException as e:
```

Comment:
I'd think in this flow, we want to catch any and all exceptions here to prevent the script dying in unforeseen circumstances.